microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/jdt/internal/core/project/PropertiesConfigSource.java

```diff
@@ -43,4 +44,14 @@ protected Properties loadConfig(InputStream input) throws IOException {
 		return properties;
 	}
 
+	@Override
+	protected Set<String> getPropertyKeys(Properties properties) {
+		return properties.stringPropertyNames();
+	}
+
+	@Override
+	public String getName() {

```

Comment:
I will remove this getter name and just use getConfigFileName. For the hover we don't need to display "Properties File: ". We need to gain some places (because path could be long),so I think we don't need the "Properties File: " information in the hover.